submissions/moyinoluwaAdenuga/todo-app/src/Components/form.js

```diff
@@ -0,0 +1,45 @@
+import React, {Component} from 'react'
+
+class Form extends Component {
+    state = {
+        input: ''
+    }
+
+    handleChange = (e) => {
+        this.setState({input: e.target.value})
+    }
+
+    handleSubmit = (e) => {
+        e.preventDefault()
+        
+        if (this.state.input === '') {
+            return false
+        }
+
+        this.props.onSubmit({
+            text: this.state.input, 
+            completed: false, 
+            id: Math.random() * 1000, 
```

Comment:
This is not good practice. You can not be setting the this way.

Reply:
@MoyinoluwaA Kindly work on this

Reply:
@ToluAkin I already did. That is the previous commit. The recent one has the corrected changes

Reply:
Which recent ones? This branch would be updated your changes if it has been corrected.

Reply:
Okay. i see the miscommunication. Don't use `Math.Random` as Id. Use the position of the todo in an array.